webpack.config.js

```diff
@@ -34,14 +36,17 @@ module.exports = {
 		} ),
 		new AppCachePlugin(),
 		new HtmlWebpackPlugin( {
-			title: 'Simplenote',
-			templateContent: require( './index-builder.js' ),
-			inject: false
+			title: 'Simplenote'
 		} ),
 		new webpack.DefinePlugin( {
 			'process.env.NODE_ENV': JSON.stringify(
 				process.env.NODE_ENV || 'development'
-			)
+			),
+			config: JSON.stringify( config() )
```

Comment:
I like embedding the state of git during the build into the product. There are many ways of doing this. One way would be making a config script that shells out and runs `git describe --always --dirty` and uses that as the version.

Mine right now: `v1.0.6-26-g40890ca-dirty`